src/widgets/Grabber.h

```diff
@@ -46,15 +46,13 @@ class GrabberEvent:public wxCommandEvent
    GrabberEvent(wxEventType type = wxEVT_NULL,
                 wxWindowID winid = 0,
                 const wxPoint& pt = wxDefaultPosition)
-   : wxCommandEvent(type, winid)
+   : wxCommandEvent(type, winid), mPos(pt)
```

Comment:
Probably because gcc can automatically optimize it for integers, but on more complex objects, this can slightly increase performance, see http://www.tantalon.com/pete/cppopt/asyougo.htm#UseConstructorInitializationLists